gateway-service/src/main/resources/gateway-log-messages.yml

```diff
@@ -266,3 +266,17 @@ messages:
       reason: "An error occurred in the SAF Auth Service. Review the reason in the error message."
       action: "Supply a valid user and application name, and check that corresponding permissions have been set up."
 
+    # IDT messages (150 - 159)
+    - key: org.zowe.apiml.security.idt.failed
+      number: ZWEAG150
+      type: ERROR
+      text: "SAF IDT generation failed. Reason: %s"
+      reason: "An error occurred in the SAF Verify Service. Review the reason in the error message."
+      action: "Verify the Identity Token configuration."
+
+    - key: org.zowe.apiml.security.idt.auth.failed
+      number: ZWEAG151
+      type: ERROR
+      text: "SAF IDT is not generated because authentication or authorization failed. Reason: %s"
+      reason: "The SAF Verify Service rejected the user credentials. Review the reason in the error message."
```

Comment:
`SAF Verify Service` - same as above

Reply:
Based on the discussion with Andrew he agreed that the proposed wording is better.